lighthouse-cli/test/smokehouse/smoke-test-dfns.js

```diff
@@ -39,7 +39,7 @@ const smokeTests = [{
 }, {
   id: 'dbw',
   expectations: require('./dobetterweb/dbw-expectations.js'),
-  config: require('./dbw-config.js'),
+  config: require('../../../lighthouse-core/config/default-config.js'),
```

Comment:
based on https://github.com/GoogleChrome/lighthouse/pull/9765#discussion_r330628803 it seemed time to just embrace it for dbw_tester and have it load them all.

Only adds like 2 extra seconds for the offline pass (which was skipped before but now quickly does nothing)